src/Components/ArticleComments.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react"
+import { fetchCommentsByArticle } from "../../utils";
+import { useParams } from "react-router";
+import moment from "moment";
+
+function ArticleComments() {
+
+   const [articleComments, setArticleComments] = useState([])
+   const {article_id} = useParams()
+   
+
+   useEffect(() => {
+       fetchCommentsByArticle(article_id).then(({comments}) => {
+        setArticleComments(comments)
+       })
+   }, [])
+
+   return (
+    <>
+    <ul className = 'comments'>
+    {articleComments.map(({comment_id, body, author, votes, created_at}) => {
```

Comment:
Could we add something here for if there are no comments? We'd like the user to receive a message giving them this information. 🙂